data/links.json

```diff
@@ -47,6 +47,11 @@
         "href": "https://alumnimap.csh.rit.edu",
         "icon": "material:language"
       },
+      {
+        "name": "Eval Database",
```

Comment:
I think it would be nice to change this to say Conditional? (Also in other places it's referenced).

Maybe "Conditonal (Evals Database)" would be the best, although I'm not sure how well that fits.

Reply:
I think everyone knows what Conditional is. I think we can safely change the name from "Evals daatabase" to "Conditional"